snapshot.go

```diff
@@ -164,13 +164,7 @@ func DiffPrettyText(diffs []diffmatchpatch.Diff) string {
 
 		switch diff.Type {
 		case diffmatchpatch.DiffInsert:
-			for _, c := range text {
-				if unicode.IsSpace(c) {
```

Comment:
Don't delete this if branch.
Just set foreground color on L171.

Reply:
done